### DISCUSSION TITLE
Error when trying to commit changes - SOLVED

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#191] [Archive] [answered]
Hi @IBAHCM/rpir-instructors 

I am trying to commit changes to GitHub and I keep getting this error message. I haven't ran anything different to any other session and never had this before. Unsure what I need to do!

<img width="702" alt="Screenshot 2021-02-01 at 16 36 32" src="https://user-images.githubusercontent.com/77625039/106488753-f060e800-64ab-11eb-91c1-cff8b0242f47.png">

Sarah

**Comment by soniamitchell** [comment, marked as answer]
Glad you got it sorted 👍
